src/store/reducers/storage/selectors.ts

```diff
@@ -72,6 +77,38 @@ export const selectNodesUptimeFilter = (state: StorageStateSlice) =>
     state.storage.nodesUptimeFilter;
 export const selectStorageType = (state: StorageStateSlice) => state.storage.type;
 
+// ==== Sort params selectors ====
+export const selectNodesSortParams = (state: StorageStateSlice) => {
+    const defaultSortValue: NodesSortValue = NODES_SORT_VALUES.NodeId;
+    const defaultSortOrder: OrderType = ASCENDING;
+
+    return {
+        sortValue: state.storage.nodesSortValue || defaultSortValue,
+        sortOrder: state.storage.nodesSortOrder || defaultSortOrder,
+    };
+};
+
+export const selectGroupsSortParams = (state: StorageStateSlice) => {
+    const visibleEntities = state.storage.visible;
+
+    let defaultSortValue: StorageSortValue = STORAGE_SORT_VALUES.PoolName;
+    let defaultSortOrder: OrderType = ASCENDING;
+
+    if (visibleEntities === VISIBLE_ENTITIES.missing) {
+        defaultSortValue = STORAGE_SORT_VALUES.Degraded;
+        defaultSortOrder = DESCENDING;
+    }
+
+    if (visibleEntities === VISIBLE_ENTITIES.space) {
+        defaultSortValue = STORAGE_SORT_VALUES.Usage;
+        defaultSortOrder = DESCENDING;
+    }
```

Comment:
These default values previously was set in `StorageGroups`.

Changed `UsedSpaceFlag` to `Usage` since `UsedSpaceFlag` cannot be sorted via request params